www/calendar/inner.js

```diff
@@ -775,7 +775,7 @@ define([
         var data = APP.calendars[id];
         var edit;
         if (data.loading) {
-            edit = h('i.fa.fa-spinner.fa-spin');
+            edit = h('i.fa.fa-spinner.fa-spin', {'aria-hidden': 'true'});
```

Comment:
This doesn't look related to the initial goal of the PR, which was to fix a UI issue on small screens.

Reply:
Some of these change with aria and adding UI elements have been requested in a previous UI review